tools/python/util/android/android.py

```diff
@@ -118,10 +126,12 @@ def start_emulator(
             "America/Los_Angeles",
             "-no-snapstorage",
             "-no-audio",
+            "-no-snapshot",  # No bluetooth
```

Comment:
I thought this parameter was about fast startup from previous run.

https://developer.android.com/studio/run/emulator-snapshots

Reply:
Yeah, that is correct. This is not needed for now.